### ISSUE TITLE
Time Zone Consistency

### ISSUE THREAD

**Issue by jzstern** [philipjkim/vue2-flip-countdown#17]
Thank you for making this package!

Realized that the countdown is relative to the time zone of the client, rather than a standardized time(e.g. UTC or Epoch). For anyone looking for a quick workaround to getting the countdown to sync up across time zones, here's my hacky solution:

```
let startTimeEpoch = 1593540000;
let d = new Date(0);
d.setUTCSeconds(startTimeEpoch);

let month = d.getMonth() + 1;
let day = d.getDate();

if (month < 10) month = "0" + month;
if (day < 10) day = "0" + day;

this.deadlineLocal =
d.getFullYear() + "-" + month + "-" + day + " " + d.getHours() + ":00:00";
```

Kinda pressed on time at the moment, but could be nifty to add an optional prop to the component that accepts an Epoch timestamp and converts it based on the client's location 😊

**Comment by fabiom91** [comment]
I agree, time zone support would make a huge difference! Thank you :)
